src/js/components/Beacon/BeaconQueryUi.tsx

```diff
@@ -17,33 +17,61 @@ import {
   BUTTON_AREA_STYLE,
   BUTTON_STYLE,
 } from '@/constants/beaconConstants';
-import { DEFAULT_TRANSLATION } from '@/constants/configConstants';
-
+const { Text, Title } = Typography;
 // TODOs
-// form verification
 // example searches, either hardcoded or configurable
-// better help text (#1691)
+// more intuitive variants ui
 
-const VARIANTS_HELP =
-  'Variants search requires the fields Chromosome, Variant start, Assembly ID, and at least one of Variant end or Alternate base(s).';
-const METADATA_HELP = 'Search over clinical or phenotypic properties.';
 const STARTER_FILTER = { index: 1, active: true };
 
+// complexity of instructions suggests the form isn't intuitive enough
+const VARIANTS_INSTRUCTIONS_TITLE = 'Variant search';
+const VARIANTS_INSTRUCTIONS_LINE1 =
+  'To search for all variants inside a range: fill both "Variant start" and "Variant end", all variants inside the range will be returned. You can optionally filter by reference or alternate bases.';
```

Comment:
i guess bento_public linting doesn't enforce this, but i'd prefer a max 120-char line width here